specs/latest/2.0-compute/index.bs

```diff
@@ -0,0 +1,1239 @@
+          OpenGL ES 3.1 Section 10.3.1</a>,
+        and <a href="https://www.khronos.org/registry/OpenGL-Refpages/es3.1/html/glVertexAttribIFormat.xhtml"
+          class="nonnormative">man page</a>
+      </span>
+  </p>
+  <p>
+    <b>
+      void vertexAttribBinding(GLuint attribindex, GLuint bindingindex)
+    </b>
+  </p>
+  <p>
+    Set the association between a vertex attribute and the vertex buffer binding.
+    <br/>See glVertexAttribBinding at
+      <span class="gl-spec">
+        <a href="https://www.khronos.org/registry/OpenGL/specs/es/3.1/es_spec_3.1.pdf#nameddest=section-10.3.1">
+          OpenGL ES 3.1 Section 10.3.1</a>,
+        and <a href="https://www.khronos.org/registry/OpenGL-Refpages/es3.1/html/glVertexAttribBinding.xhtml"
+          class="nonnormative">man page</a>
+      </span>
+  </p>
+  <p>
+    <b>
+      void vertexBindingDivisor(GLuint bindingindex, GLuint divisor)
+    </b>
+  </p>
+  <p>
+    Set the advisor value for attributes taken from the buffer bound to bindingindex between a vertex attribute and the vertex buffer binding.
+    <br/>See glVertexBindingDivisor at
+      <span class="gl-spec">
+        <a href="https://www.khronos.org/registry/OpenGL/specs/es/3.1/es_spec_3.1.pdf#nameddest=section-10.3.2">
+          OpenGL ES 3.1 Section 10.3.2</a>,
+        and <a href="https://www.khronos.org/registry/OpenGL-Refpages/es3.1/html/glVertexBindingDivisor.xhtml"
+          class="nonnormative">man page</a>
+      </span>
+  </p>
+
+### Writing to the Drawing Buffers ### {#drawing}
+  <p>
+    <b>
+      void drawArraysIndirect(GLenum mode, GLintptr offset)
+    </b>
+  </p>
+  <p>
+    Draw geometry with vertices by reading parameters via indirect buffer.
+    <br/>See glDrawArraysIndirect at
+      <span class="gl-spec">
+        <a href="https://www.khronos.org/registry/OpenGL/specs/es/3.1/es_spec_3.1.pdf#nameddest=section-10.5">
+          OpenGL ES 3.1 Section 10.5</a>,
+        and <a href="https://www.khronos.org/registry/OpenGL-Refpages/es3.1/html/glDrawArraysIndirect.xhtml"
+          class="nonnormative">man page</a>
+  </p>
+  <p>
+    <b>
+      void drawElementsIndirect(GLenum mode, GLenum type, GLintptr offset)
+    </b>
+  </p>
+  <p>
+    Draw geometry with indexed vertices by reading parameters via indirect buffer.
+    <br/>See glDrawElementsIndirect at
+      <span class="gl-spec">
+        <a href="https://www.khronos.org/registry/OpenGL/specs/es/3.1/es_spec_3.1.pdf#nameddest=section-10.5">
+          OpenGL ES 3.1 Section 10.5</a>,
+        and <a href="https://www.khronos.org/registry/OpenGL-Refpages/es3.1/html/glDrawElementsIndirect.xhtml"
+          class="nonnormative">man page</a>
+      </span>
+  </p>
+
+### Program Interface Query ### {#program-interface}
+  <p>
+    <b>
+      any getProgramInterfaceParameter(WebGLProgram? program, GLenum programInterface, GLenum pname)
```

Comment:
It doesn't seem to be necessary to make `program` nullable here. Other methods in this spec may have the same issue.

Reply:
Done.